### ISSUE TITLE
KIF waitForViewWithAccessibilityLabel does not match patterns having % correctly

### ISSUE THREAD

**Issue by XPCD** [kif-framework/KIF#688]
Hi,

I am trying to wait for a label (using waitForViewWithAccessibilityLabel) having a pattern of the type -

SOMETHING, SOMETHING, New, New, ok, ok, 40%, 20%, 5°, 10° 

My NSLog shows the exact pattern when I print the variable. However, KIF throws the following error message -

Failed to find accessibility element with the label "SOMETHING, SOMETHING, New, New, ok, ok, 40, 20, 5°, 10°" 

Looks like the two '%' signs are not picked up for matching the pattern. This seems to be inconsistent behaviour because a couple of other tests work fine. I have checked the white spaces and string length many times for this issue but there is no difference.

**Comment by phatmann** [comment]
See #692.
